pxr/usd/bin/usdedit/usdedit.py

```diff
@@ -87,7 +86,7 @@ def _generateTemporaryFile(usdcatCmd, usdFileName, readOnly, prefix):
 def _editTemporaryFile(editorCmd, usdaFileName):
     # check the timestamp before updating a file's mtime
     initialTimeStamp = os.path.getmtime(usdaFileName)
-    os.system(editorCmd + ' ' + usdaFileName)
+    subprocess.call([editorCmd, usdaFileName])
```

Comment:
This seems like a great change to make.  One note-- I think `subprocess.run` is preferred over `subprocess.call` for new usages.

https://docs.python.org/3/library/subprocess.html#subprocess.run